m365-developer-proxy/Program.cs

```diff
@@ -17,4 +17,41 @@
 
 rootCommand.Handler = proxyHost.GetCommandHandler(pluginEvents, loaderResults.UrlsToWatch, logger);
 
-return await rootCommand.InvokeAsync(args);
\ No newline at end of file
+// store the global options that are created automatically for us
+string[] globalOptions = { "--version", "-?", "-h", "--help" };
```

Comment:
Seems like we're missing a few accepted variants: https://github.com/dotnet/command-line-api/blob/a045dd54a4c44723c215d992288160eb1401bb7f/src/System.CommandLine/Help/HelpOption.cs#L25

Let's add them

Reply:
Done